pkg/build/pipelines/python/import.yaml

```diff
@@ -12,16 +12,51 @@ inputs:
   import:
     description: |
       The package to import.
-    required: true
+    required: false
+  imports:
+    description: |
+      List of packages to import, separated by spaces.
+    required: false
   from:
     description: |
-      The package to import (from <from> import <import>).
+      The package to import from (used with 'from <from> import <import>').
```

Comment:
`from` is only relevant to `import`, right? lets mention that.
even lets mention `deprecated` on both of those, suggesting use of `imports`